src/schemas/json/backportrc.json

```diff
@@ -49,8 +49,8 @@
         ]
       }
     },
-    "upsteam": {
-      "description": "Identifier for the GitHub project as `{owner}/{repoName}`",
+    "upstream": {
```

Comment:
This was a fix - this deprecated field is called `upstream`, not `upsteam`. Had to check the source, as it's not documented anymore.